README.md

```diff
@@ -152,7 +152,15 @@ your app project, _if there is something that not working, please check the list
 with WeChat app.
 
 #### registerApp(appid)
+You should be the global registration, and only one time
```

Comment:
Correct it to: "You should call this function in global, calling over twice would throw an error."